tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java

```diff
@@ -202,7 +238,7 @@ private KafkaProducer<byte[], byte[]> createProducer(Properties config) {
     }
 
     private Consumer<byte[], byte[]> createConsumer(Properties config) {
```

Comment:
Could you add tests for `GroupProtocol.CONSUMER`? see comment: https://github.com/apache/kafka/pull/15766#discussion_r1576009920

Reply:
Can we do it in another PR?